src/Mapper/Fields/FieldMappingInterface.php

```diff
@@ -29,5 +30,10 @@ public function getName(): string;
      */
     public function getEntityAttribute(): ?string;
 
+    /**
+     * Dedicated service to convert the entity value to a single field value.
+     */
+    public function getFieldConverter(): ?FieldConverterInterface;
```

Comment:
1. Add into interface for BC
2. or create a sub interface
3. Use method exists

Make sure 1 is your favorite